src/Examples/Flowsheets/power_generation/rsofc/rsofc_costing.py

```diff
@@ -0,0 +1,544 @@
+###############################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University,
+# West Virginia University Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+###############################################################################
+
+"""
+Costing methods for the soec and sofc operating modes of the reversible sofc
+
+"""
+__author__ = "Chinedu Okoli", "Alex Noring"
+
+import pyomo.environ as pyo
+from pyomo.environ import units as pyunits
```

Comment:
You should just use `pyo.units` instead here (seeing as you just imported all of `pyomo.environ` above.